fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/group/ChunkReader.java

```diff
@@ -85,10 +85,13 @@ private Group findGroupByID(String groupId) throws Exception{
         searchContext = SearchUtil.parseQueryParameters(Group.class, queryParameters);
         List<Resource> resources = null;
         FHIRTransactionHelper txn = new FHIRTransactionHelper(fhirPersistence.getTransaction());
-        txn.enroll();
-        persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
-        resources = fhirPersistence.search(persistenceContext, Group.class).getResource();
-        txn.unenroll();
+        txn.begin();
+        try {
+            persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
+            resources = fhirPersistence.search(persistenceContext, Group.class).getResource();
+        } finally {
+            txn.end();
+        }
```

Comment:
not sure if this can work well, need to be verified by the minio enabled build pipeline.